packages/flutter_tools/test/daemon_test.dart

```
@@ -147,7 +134,7 @@ void main() {
       commands.close();
     });
 
-    _testUsingContext('daemon.restart', () async {
+    testUsingContext('daemon.restart', () async {
```

"app.restart without an appId should report an error"